src/main/java/dev/ftb/mods/ftbteambases/config/ServerConfig.java

```diff
@@ -112,6 +133,58 @@ static int getNetherPortalYPos(Player player) {
         return USE_CUSTOM_PORTAL_Y_POS.get() ? CUSTOM_PORTAL_Y_POS.get() : player.blockPosition().getY();
     }
 
+    static Color4I getLobbyTeamColor() {
+        Color4I teamColor = Color4I.fromString(ServerConfig.LOBBY_CLAIM_COLOR.get());
+        return teamColor.isEmpty() ? Color4I.rgb(0xFF40FF) : teamColor;
+    }
+
+    static ChunkPos getClaimCenter() {
+        int[] pos = ServerConfig.LOBBY_CLAIM_CENTER.get();
+        if (pos.length == 2) {
+            return new ChunkPos(pos[0], pos[1]);
+        } else {
+            FTBTeamBases.LOGGER.error("invalid lobby claim centre pos! expected 2 integers, got {}. default to (0, 0)_", pos.length);
+            return new ChunkPos(0, 0);
+        }
+    }
+
+    enum AutoClaimShape {
```

Comment:
As this is more than just a simple enum, I feel like it deserves it's own class file seeing as it's also being used in the `AutoClaiming` class